hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_main_sim.core

```diff
@@ -20,8 +20,21 @@ filesets:
 
 
 targets:
-  sim:
+  sim: &sim_target
     toplevel: xbar_tb_top
     filesets:
       - files_dv
     default_tool: vcs
+
```

Comment:
Ohh, I missed this. This is autogenerated file (below one too;  template is in the tlgen area I think). Would you please prepare a quick patch?